rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1070,8 +1070,7 @@ public boolean hasBtcBlockCoinbaseTransactionInformation(Object[] args) {
         return (self, args) -> {
             Federation retiringFederation = self.bridgeSupport.getRetiringFederation();
 
-            if (!BridgeUtils.isFromFederateMember(self.rskTx, self.bridgeSupport.getActiveFederation())
-                    && ( retiringFederation == null || (retiringFederation != null && !BridgeUtils.isFromFederateMember(self.rskTx, retiringFederation)))) {
+            if (!BridgeUtils.isFromFederateMember(self.rskTx, self.bridgeSupport.getActiveFederation()) && (retiringFederation == null || !BridgeUtils.isFromFederateMember(self.rskTx, retiringFederation))) {
```

Comment:
Please indent the code to increase readability

Reply:
Done!